### PR TITLE
Kinesis variable update

### PR DESCRIPTION
**Pull request by resdenia**

### CODE REVIEW

File: python3/kinesis/src/lambda_function.py [hunk 1]
```diff
@@ -96,11 +97,16 @@ def split_by_fields(log, field):
...
+
 def lambda_handler(event, context):
     # type: (dict, 'LambdaContext') -> None
-    logger.info("Received {} raw Kinesis records.".format(len(event["Records"])))
+    logger.info("Received {} raw Kinesis records.".format(
+        len(event["Records"])))
     multiple_msgs = os.environ.get(MESSAGES_ARRAY_ENV)
 
+    if not os.environ.get('COMPRESS'):
+        os.environ['COMPRESS'] = "True"
```
[the remark this answers is not in the document]
Comment (resdenia) [reply python3/kinesis/src/lambda_function.py:108]: @mirii1994 
I'm not sure that is a right suggestion. cause in a code with a shipper(shipper.py) we have only
```
        try:
            self._compress = os.environ['COMPRESS'].lower() == "true"
        except KeyError as e:
            self._compress = False
```

[the remark this answers is not in the document]
Comment (mirii1994) [reply python3/kinesis/src/lambda_function.py:108]: `str(True)` will produce the string: `"True"`. It's basically the same as your line, but just uses some python functionality. Then the code in the shipper will convert it to `"true"` as it uses the `lower()` function.

[the remark this answers is not in the document]
Comment (resdenia) [reply python3/kinesis/src/lambda_function.py:108]: Nice! Thank you!

**Review by mirii1994 — APPROVED** [review]
Great!

**Pull request merged**